package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "ember-bulma",
-  "version": "0.0.0-semantically-released",
+  "name": "@magikcraft/ember-bulma",
```

Comment:
The name of the package shouldn't be changed. I assume this wasn't intended for this PR but carried over from your personal fork?